manifest.toml

```diff
@@ -0,0 +1,57 @@
+packaging_format = 2
+
+id = "umami"
+name = "Umami"
+description.en = "Simple, fast, privacy-focused alternative to Google Analytics"
+description.fr = "Analyse de trafic web simple et respectueux"
+
+version = "1.40.0~ynh1"
+
+maintainers = ["eric_G"]
+
+[upstream]
+license = "MIT"
+website = "https://umami.is/"
+demo = "https://app.umami.is/share/8rmHaheU/umami.is"
+admindoc = "https://umami.is/docs/about"
+code = "https://github.com/mikecao/umami"
+
+[integration]
+yunohost = ">= 11.1.7"
+architectures = "all"
+multi_instance = true
+ldap = false
+sso = false
+disk = "50M"
+ram.build = "50M"
+ram.runtime = "50M"
+
+[install]
+    [install.domain]
+    type = "domain"
+    full_domain = true
+
+    [install.init_main_permission]
+    type = "group"
+    default = "visitors"
+
+[resources]
+    [resources.system_user]
+
+    [resources.install_dir]
+
+    [resources.ports]
+
+    [resources.permissions]
+    main.url = "/"
+
+    [resources.apt]
+    packages = "postgresql"
+
+    [[resources.apt.extras]]
+    repo = "deb https://dl.yarnpkg.com/debian/ stable main"
+    key = "https://dl.yarnpkg.com/debian/pubkey.gpg"
+    packages = "yarn"
```

Comment:
```suggestion
    extras.yarn.repo = "deb https://dl.yarnpkg.com/debian/ stable main"
    extras.yarn.key = "https://dl.yarnpkg.com/debian/pubkey.gpg"
    extras.yarn.packages = "yarn"
```

Reply:
Shall we change the convert script ? https://github.com/YunoHost/apps/blob/d18008456654bc2fb8aac40982664336da00ad53/tools/packaging_v2/convert_app_to_packaging_v2.py#L125-L127